installer/unix/install-cli

```diff
@@ -84,16 +78,22 @@ else
 fi
 
 # Download JSON manifest with latest CLI binary
-echo ":::: Installing the Particle CLI for $OS to \"$DEST\""
-
-BINARY_URL=$(curl -s $MANIFEST_URL | $PYTHON -c "import sys, json; print(json.load(sys.stdin)['builds']['$OS']['$ARCH']['url'])")
-BINARY_SHA256=$(curl -s $MANIFEST_URL | $PYTHON -c "import sys, json; print(json.load(sys.stdin)['builds']['$OS']['$ARCH']['sha256'])")
+echo ":::: Installing the Particle CLI for $OS $ARCH"
+MANIFEST=$(curl -s --fail $MANIFEST_URL)
```

Comment:
`--fail` causes curl to exit with error code 22 if the file can't be downloaded (HTTP 400 or above).